src/sys/wait.rs

```diff
@@ -235,6 +235,57 @@ pub fn waitpid<P: Into<Option<Pid>>>(pid: P, options: Option<WaitPidFlag>) -> Re
     }
 }
 
+
+/// The same as calling waitpid(None, None)
```

Comment:
Thanks for adding docs where they were missing.  Do you think you could turn this one into an internal link?  It would look something like [`waitpid(None, None)`](fn.waitpid.html)`.